src/main/java/com/faforever/api/data/domain/AbstractEntity.java

```diff
@@ -0,0 +1,38 @@
+package com.faforever.api.data.domain;
+
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Inheritance;
+import javax.persistence.InheritanceType;
+import javax.persistence.MappedSuperclass;
+import java.time.OffsetDateTime;
+
+@MappedSuperclass
+@Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
```

Comment:
`@MappedSuperclass` is for `JPA`
and `@Inhertiance` defines the Strategy and we need `TABLE_PER_CLASS` from my understanding ...
I will test it ...